core/src/filter/resources/jenkins/model/jenkins-version.properties

```diff
@@ -1 +1,2 @@
-version=${build.version}
\ No newline at end of file
+version=${build.version}
+java.min.level=${java.level}
```

Comment:
@oleg-nenashev If I was my choice I would remove `jenkins-version.properties`. IMHO the right place is the standard file: `MANIFEST`.

I'm not sure if `jenkins-version.properties` was created for including other properties, at least, the name does not tell that.